enterprise_access/apps/api/v1/views/content_assignments/assignments_admin.py

```diff
@@ -204,3 +204,32 @@ def cancel(self, request, *args, uuid=None, **kwargs):
             return Response(response_serializer.data, status=status.HTTP_200_OK)
         else:
             return Response(status=status.HTTP_422_UNPROCESSABLE_ENTITY)
+
+    def remind(self, request, *args, uuid=None, **kwargs):
+        """
+        Send reminders to a single learners with associated ``LearnerContentAssignment``
+        record by uuid.
+
+        Raises:
+            404 if the assignment was not found.
+            422 if the assignment was not able to be reminded.
+        """
+        try:
+            assignment_to_remind = self.get_queryset().get(uuid=uuid)
+        except LearnerContentAssignment.DoesNotExist:
+            return Response(None, status=status.HTTP_404_NOT_FOUND)
+
+        # if the assignment is not remindable, this is a no-op.
+        reminder_info = assignments_api.remind_assignments([assignment_to_remind])
```

Comment:
I think this view (and the cancel view, too) have to support bulk actions from the admin portal UI, meaning they have to support the cancellation and reminder of multiple assignments in a single request.  So we'll want this to be a DRF action with `detail=False` that expects to have a list of assignment UUIDs in the POST payload to remind (or cancel for the `cancel` view).

Reply:
Don't worry about changing the `cancel` view for your PR - I'll ticket out some additional work to fix that up.